pkg/genericapiserver/options/server_run_options.go

```diff
@@ -159,6 +159,7 @@ func NewServerRunOptions() *ServerRunOptions {
 func (o *ServerRunOptions) WithEtcdOptions() *ServerRunOptions {
 	o.StorageConfig = storagebackend.Config{
 		Prefix: DefaultEtcdPathPrefix,
+		Quorum: DefaultReadQuorumFlag,
```

Comment:
@jszczepkowski It was not efficient pre-etcd 3.1 release. q-read is effectively noop + read for one node cluster since etcd 3.1. release. q-read is also almost as fast as normal read for multiple nodes etcd cluster when network latency < 1ms, which is the case on GCE. Basically, there is no reason to turn it off soon.

@wojtek-t We will release 3.1 in Dec. For 1.6 release, we will upgrade etcd to 3.1 directly I assume. So this wont be a problem.